tests/topotests/bfd_vrflite_topo1/r1/bfdd.conf

```diff
@@ -0,0 +1,16 @@
+!
+! debug bfd network
+! debug bfd peer
+! debug bfd zebra
+!
+bfd
+ peer 192.168.0.2 vrf vrf1
+ !
+ peer 192.168.0.2 vrf vrf2
+ !
+ peer 192.168.0.2
+ !
+ peer 2.2.2.2 multihop local-address 1.1.1.1 vrf vrf1
```

Comment:
Can we avoid Cloudflare? :)

Reply:
1.1.1.1 and 2.2.2.2  are used a lot on topotests. Did I miss something in the remark ?